client/src/main/java/com/distkv/dst/client/DstSortedListProxy.java

```diff
@@ -53,21 +53,21 @@ public void incrItem(String key, String member, int delta) {
     topRequestBuilder.setCount(topNum);
     SortedListProtocol.TopResponse response = FutureUtils.get(
         service.top(topRequestBuilder.build()));
-    checkException(response.getStatus(),key);
+    checkStatus(response.getStatus(),key);
```

Comment:
```suggestion
    checkStatus(response.getStatus(), key);
```

And the same other issues.